README.md

```diff
@@ -56,6 +57,7 @@ logzio/docker-collector-logs
 Spin up your Docker containers if you haven’t done so already. Give your logs a few minutes to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
 
 ### Change log
+- 0.1.2: Add rename processor ("renameFields") to specify a list of fields to rename.
```

Comment:
In the script, there is a variable called `DOCKER_COLLECTOR_VERSION` and is currently set to `0.1.1`. Please make sure whenever you're working on a new version to update it to the right version. This variable is logging the version at the beginning of the script so it's important that it will match the version.